src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -0,0 +1,103 @@
+---
+group: cloud-guide
+title: Magento Cloud Docker release notes
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+The [Magento Cloud Docker](https://github.com/magento/magento-cloud-docker) package provides functionality and Docker images to deploy {{ site.data.var.ee }} to a local Cloud environment. These release notes describe the latest improvements to this package, which is a component of [{{ site.data.var.csuite }}]({{ page.baseurl }}/cloud/release-notes/cloud-tools.html).
+
```

Comment:
Add link to Backward incompatible changes topic: https://devdocs.magedevteam.com/1381/cloud/release-notes/backward-incompatible-changes.html